src/ParticularTemplates/ParticularTemplates.csproj

```diff
@@ -5,6 +5,7 @@
     <Description>dotnet new templates targeting Particular tools and libraries</Description>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <PackageType>Template</PackageType>
+    <LangVersion>latest</LangVersion>
```

Comment:
There is code in this project due to:

```C#
  <ItemGroup>
    <None Include="..\Templates\**\*.*" Exclude="**\bin\**\*.*;**\obj\**\*.*" Pack="true" PackagePath="content" />
  </ItemGroup>
```

Indeed, when you open the solution, this is the project that is opened.

I added it because I wanted to use `async main` etc. but I'll address that consideration in response to your other comment.

Reply:
There is no compiled code. Note that that include is for the `None` group.

Reply:
I missed that. 😄 

OK, in that case I'll remove this element, since it's serving no purpose.